helpers/import.js

```diff
@@ -0,0 +1,15 @@
+function importHelper(type, file) {
+  switch (process.env.moduleFormat) {
+    case "esm":
+      return `import ${type} from "./${file}.js"`;
+    case "cjs":
```

Comment:
Personally I think I'd prefer longer option names, e.g. "esmodule" or "commonjs", but I'm not too fussed :)

Reply:
I think that's a fair point. I've updated with the long names